File: Python Version/VS Code py Format/coefficientprolate.py
#Hamed Baghal Ghaffari
#2021-10-17
#This Function gives us the eigenvectors of the prolatematrix after sorting the eigenvalues.
from numpy import linalg

from Prolatematrix import Prolatematrix

def coefficientprolate(c,m,n):
    A=Prolatematrix(c,m)
    eigenValues, eigenVectors = linalg.eig(A)
    idx = eigenValues.argsort()[::+1]   
    eigenValues = eigenValues[idx]
    eigenVectors = eigenVectors[:,idx]
    return eigenVectors[:,n]

#print(coefficientprolate(1,4,1))

File: Julia Version/coefficientprolate.jl
#Hamed Baghal Ghaffari
#2021-10-20
#This Function gives us the eigenvectors of the prolatematrix after sorting the eigenvalues.
function coefficientprolate(c,m,n)
        A=prolatematrix(c,m);
        V=eigvecs(A);
        W=V[:,n];
        return W
end

using LinearAlgebra
